pkg/config/static_manager_test.go

```diff
@@ -57,7 +58,7 @@ func TestStaticGetOptimizelyConfig(t *testing.T) {
 
 	optimizelyConfig := configManager.GetOptimizelyConfig()
```

Comment:
I think you should be calling the functions getAttributes and getEvents and compare those results to the expected based on the provided datafile. Looks liek coverage isnt getting hit for those two functions still..

Reply:
Sorry this is for testing config.go

Reply:
Putting a panic there triggers on `make test`. Need to investigate.

Reply:
They get called when loading, and it's not showing up in coverage for some reason. I can test the values later, but need to debug the coveralls.